utils.html

```diff
@@ -44,3 +45,7 @@ <h3 class="s-h3">{{ border_group | capitalize}}</h3>
   {% assign utils = border_utils | where: "border_group", border_group %}
   {% include utils-table.html utils=utils table_class="utils-table-borders" %}
 {% endfor %}
+
+<h2 id="shadows" class="s-h2 s-mt-5">Shadows</h2>
+{% assign shadow_utils = site.utils | where: "util_type", "shadow" %}
+{% include utils-table.html utils=shadow_utils table_class="utils-table-padding" %}
```

Comment:
Do we need the `utils-table-padding` class here? If so, we should probably rename that class to something more abstract and not explicitly tied to padding.

Reply:
It has a few things that are unique to it to illustrate the padding, and margin has a few that are unique to it, too, so I think that's why this pattern started?